okhttp-tests/src/test/java/com/squareup/okhttp/UrlComponentEncodingTester.java

```diff
@@ -280,28 +282,31 @@ public String encode(int codePoint) {
     },
     PATH {
       @Override public String urlString(String value) {
-        return "http://example.com/" + value;
+        return "http://example.com/a" + value + "z/";
```

Comment:
I guess technically you don't need it here either... but it at least makes sense here.